changelogs/fragments/58323-copy-deep-recursive-with-remote_src.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+- copy - recursive copy with remote_src=yes now recurses beyond first level. Fixes (https://github.com/ansible/ansible/issues/58284)
```

```suggestion
- copy - recursive copy with ``remote_src=yes`` now recurses beyond first level. (Fixes https://github.com/ansible/ansible/issues/58284)
```
(Or remove the brakets completely.)